manage_breast_screening/participants/models/symptoms.py

```diff
@@ -0,0 +1,51 @@
+from django.db import models
+
+from ...core.models import BaseModel, ReferenceDataModel
+from .appointment import Appointment
+from .participant import Participant
+
+
+class SymptomType(ReferenceDataModel):
+    name = models.CharField(null=False, blank=False)
+
+
+class SymptomSubType(ReferenceDataModel):
+    symptom_id = models.ForeignKey(SymptomType, on_delete=models.CASCADE)
+    name = models.CharField(null=False, blank=False)
+
+
+class SymptomAreas(models.TextChoices):
+    RIGHT_BREAST = ("RIGHT_BREAST", "Right")
+    LEFT_BREAST = ("LEFT_BREAST", "Left")
+    BOTH_BREASTS = ("BOTH_BREASTS", "Both")
+    OTHER = ("OTHER", "Other")
+
+
+class ParticipantReportedSymptom(BaseModel):
+    symptom_type_id = models.ForeignKey(SymptomType, on_delete=models.PROTECT)
+    symptom_sub_type_id = models.ForeignKey(
+        SymptomSubType, on_delete=models.PROTECT, null=True, blank=True
+    )
+    participant_id = models.ForeignKey(Participant, on_delete=models.PROTECT)
+    appointment_id = models.ForeignKey(Appointment, on_delete=models.PROTECT)
+    reported_at = models.DateField()
+
+    # Optional further description of the symptom
+    description = models.CharField(blank=True, null=False)
+
+    # Whether the symptom is associated with the right/left breast
+    area = models.CharField(choices=SymptomAreas, blank=False, null=False)
+    area_description = models.CharField(blank=True, null=False)
+
+    # Has the symptom been investigated already?
+    investigated = models.BooleanField()
+
+    # Onset information
+    when_started = models.CharField(blank=True, null=False)
+    year_started = models.IntegerField(null=True, blank=True)
+    month_started = models.IntegerField(null=True, blank=True)
```

Comment:
Alternatively we could use a [DateRange](https://docs.djangoproject.com/en/5.2/ref/contrib/postgres/fields/#daterangefield) for this. But it seems a bit more complex.